Add test_mimi to ci

iseeyuan opened the pull request:
### Summary
Add test_mimi to ci

### Test plan
ci

iseeyuan commented:
@pytorchbot label "topic: not user facing"

jackzhxng commented on .github/workflows/pull.yml:
```diff
@@ -290,6 +290,33 @@ jobs:
         PYTHON_EXECUTABLE=python bash .ci/scripts/test_llava.sh
 
+  test-moshi-linux:
+    name: test-llava-runner-linux
```
Still says Llava

iseeyuan replied:
Good catch!

jackzhxng reviewed (commented):
Even though Llava is already in pull, I would prefer not having these large tests in pull, trunk would be better.

Commit:
Add test_mimi to ci (#9530)

Summary:
Add test_mimi to ci

Pull Request resolved: https://github.com/pytorch/executorch/pull/9530

Test Plan: ci

Reviewed By: billmguo

Differential Revision: D71699919

Pulled By: iseeyuan
